templates/macros/datatables.html.twig

```diff
@@ -74,6 +74,7 @@
                             {% if title != 'actions' and not headerOptions.orderBy is same as(false) %}
                                 {% if orderBy == headerOptions.orderBy %}
                                     {% set headerClass = headerClass ~ ' sortable sorting_' ~ (order) %}
+                                    {% set ariaSort = order ~ 'ending' %}
```

Comment:
Isn't there a space before the `ending` missing? What does `ending` mean?
Have you tried that on any page? What happens if order == false?

Reply:
The `aria-sort` property can have two values: `ascending` and `descending`. We already have similar wordings in the class, `sorting_asc` and `sorting_desc`. This is converting `asc` into `ascending` by adding an appropriate `ending` :)

I you prefer I can try to rewrite it in a more easy to understand way.

Reply:
Oha, I didn't understand that. A bit too magical I'd say. 

One of my issues is the next major release which is right before the door and which needs merging: 
https://github.com/kevinpapst/kimai2/pull/2902

Can you refactor to something like `if 'desc' in order` please.

Reply:
Fixed in 98c3498e2aadf11b8df71ffc11e19723ae23776c

Reply:
I will make a second MR that targets v2. I'm sorry I was not aware that this was being worked on, but it is looking great!